src/Sulu/Bundle/TestBundle/Testing/SuluTestCase.php

```diff
@@ -210,12 +177,41 @@ protected function purgeDatabase()
     }
 
     /**
-     * Shuts the kernel down if it was used in the test.
+     * @deprecated Use the container to get these services instead.
+     *
+     * TODO: Remove this and all references to it, then remove the
+     *       symfony-cmf/testing component from composer.json.
      */
-    protected static function ensureKernelShutdown()
+    public function getDbManager($type)
```

Comment:
This and the method below are from the CMF test class, and are pretty useless proxies to the container services. Removing all references to them is possible. We would not break our BC promise with the removal of these methods.